.github/workflows/aws-terragrunt-apply.yaml

```diff
@@ -36,7 +36,6 @@ jobs:
       contents: read  # This is required for actions/checkout
       actions: write
       packages: read # This is required for the workflow to pull a container
-      pull-requests: write
```

Comment:
This permission was needed in aws-terragrunt-pr-plan b/c create a comment on the pr with a terragrunt plan summary. This is not needed in the apply workflow